### PR TITLE
[RFR] Update Extra Tools documentation

### PR DESCRIPTION
**Pull request by oxocode**
* Fixes a few grammatical errors in the [SassDoc Extra tools](http://sassdoc.com/extra-tools/) documentation.
* Also fixes a spelling error in the first line.

### CODE REVIEW

File: extra-tools/index.md [hunk 1]
```diff
@@ -20,7 +21,7 @@ module.exports = function (dest, ctx) {
 };
 {% endhighlight %}
 
-If you use multiple extras, you can "chain" them like this:
+To use multiple extras, place the extras in an array:
```
Comment (pascalduez) [inline extra-tools/index.md]: Actually it's not an Array, just function parameters.

File: extra-tools/index.md [hunk 2]
```diff
@@ -28,15 +29,17 @@ extras(ctx, 'markdown', 'display', 'shortcutIcon');
...
 
 ## Display toggle (`display`)
 
-Depending on whether or not you want to display items based on their visibility (access level, `public` or `private`), the `display` filter might be able to help you. When used, it removes items that should not be displayed.
+The `display` filter allows you to display items based on their visibility (access level: `public` or `private`). This filter removes items that should not be displayed.
 
-To determine if an item should be displayed or not, this filter reads the `display.access` key from the context configuration (so make sure your theme defines one).
+The `display` filter uses the `display.access` key from the context configuration to determine if an item should be displayed or not.
+
+_**Note:** It's always best to define an access level._
```
Comment (pascalduez) [inline extra-tools/index.md]: We have a predefined style for notes: `<p class="note note--info">`.  So maybe better to stick with it.

File: extra-tools/index.md [hunk 3]
```diff
@@ -60,7 +63,9 @@ The `groupName` filter makes it possible for you to define aliases for your grou
 }
 {% endhighlight %}
 
-Note that this filter rewrites the `group` key from each item. For instance, an item with such a group key:
+_**Note:** The `groupname` filter overrides the `group` key of each item._
```
Comment (pascalduez) [inline extra-tools/index.md]: `<p class="note note--info">`

File: extra-tools/index.md [hunk 4]
```diff
@@ -68,7 +73,7 @@ Note that this filter rewrites the `group` key from each item. For instance, an
 }
 {% endhighlight %}
 
-... would look like this once the `groupName` filter has been used:
+To use the `groupName` filter:
```
Comment (pascalduez) [inline extra-tools/index.md]: The code example underneath is the result of the transformation after the filter is applied, and not an usage example.
So this is wrong.

File: extra-tools/index.md [hunk 5]
```diff
@@ -78,9 +83,9 @@ Note that this filter rewrites the `group` key from each item. For instance, an
 
 ## Shortcut icon (`shortcutIcon`)
 
-When used, the `shortcutIcon` filter takes the eponymous key from the configuration and converts it into an object with `type`, `url` and `path` keys.
+The `shortcutIcon` filter takes the eponymous key from the configuration and converts it into an object with `type`, `url` and `path` keys.
 
-For instance, `"http://absolute.path/to/icon.png"` as a value in `ctx.shortcutIcon` would yield `ctx.shortcutIcon` as:
+To use the icon URL `http://absolute.path/to/icon.png` in `ctx.shortcutIcon`:
```
Comment (pascalduez) [inline extra-tools/index.md]: The code example underneath is the result of the transformation after the filter is applied, and not an usage example.
So this is wrong.

File: extra-tools/index.md [hunk 6]
```diff
@@ -89,7 +94,7 @@ For instance, `"http://absolute.path/to/icon.png"` as a value in `ctx.shortcutIc
 }
 {% endhighlight %}
 
-On the other hand, a relative path like `"relative/path/to/icon.png"` would leave `ctx.shortcutIcon` as:
+To use the relative path `relative/path/to/icon.png` in `ctx.shortcutIcon`:
```
Comment (pascalduez) [inline extra-tools/index.md]: Same here :)

File: extra-tools/index.md [hunk 7]
```diff
@@ -125,9 +130,13 @@ The sort order is determined if the last character is either `>` or `<` (respect
...
 
-## Description
+## Description (`description`, `descriptionPath`)
+
+This filter introduces the `description` and `descriptionPath` configuration keys.
 
-The `description` filter introduces a `description` and `descriptionPath` configuration keys. The first is a raw description text, the second is a path to a file to find the description text (and will override the `description` key).
+The `description` key contains raw description text. The `descriptionPath` contains the path to a file containing the description.
+
+_**Note:** Using `descriptionPath` will override the `description` key._
```
Comment (pascalduez) [inline extra-tools/index.md]: `<p class="note note--info">`

File: extra-tools/index.md [hunk 8]
```diff
@@ -135,4 +144,6 @@ The `description` filter introduces a `description` and `descriptionPath` config
 }
 {% endhighlight %}
 
-The `descriptionPath` is relative to the configuration file, and have no required format. However the `markdown` filter will be happy to parse the description as Markdown if called **after** `description`.
+The `descriptionPath` is relative to the configuration file, and has no required format.
+
+_**Note:** If the `markdown` filter is called **after** the `description` filter, it will parse the description value as Markdown._
```
Comment (pascalduez) [inline extra-tools/index.md]: `<p class="note note--info">`

**Comment by pascalduez** [comment]
Thanks for all the corrections! 
There's a couple of mistakes on some points, could you please fix them ?  

Also waiting for another team member review.

File: extra-tools/index.md (hunk 3 above)
Comment (valeriangalliat) [inline extra-tools/index.md]: Also, camelized `groupName`.

**Comment by valeriangalliat** [comment]
Agreed with @pascalduez notes, otherwise LGTM.
Thank you @oxocode!

**Comment by oxocode** [comment]
Thanks for the PR review, @pascalduez and @valeriangalliat! I've made the changes requested and updated the branch. 
No rush, as I know you'll get to this when you can. Much appreciated!

**Comment by valeriangalliat** [comment]
I'm not sure of the "Result of using ..." format, otherwise LGTM.

File: extra-tools/index.md [hunk 9]
```diff
@@ -28,15 +29,17 @@ extras(ctx, 'markdown', 'display', 'shortcutIcon');
...
-You may have noticed most descriptions from annotations are parsed as Markdown. This is thanks to the Markdown filter from [sassdoc-extras][repo_extras] (using [Marked](https://github.com/chjj/marked)).
+You may have noticed most descriptions from annotations are parsed as Markdown. This is thanks to the Markdown filter from [sassdoc-extras][repo_extras] (using [Marked](https://github.com/chjj/marked)). This filter overrides the provided values.
 
-This filter overrides the provided values, so a value such as ``This is some `code`.`` would become ``This is some <code>code</code>``.
+For example, ``This is some `code`.`` becomes ``This is some <code>code</code>``.
 
 ## Display toggle (`display`)
 
-Depending on whether or not you want to display items based on their visibility (access level, `public` or `private`), the `display` filter might be able to help you. When used, it removes items that should not be displayed.
+The `display` filter allows you to display items based on their visibility (access level: `public` or `private`). This filter removes items that should not be displayed.
```
Comment (pascalduez) [inline extra-tools/index.md]: We should update this part, actually nothing is removed, just a new key added, so it's up to the theme to choose what to display. Not part of this PR anyway.

File: extra-tools/index.md [hunk 10]
```diff
@@ -60,7 +63,9 @@ The `groupName` filter makes it possible for you to define aliases for your grou
 }
 {% endhighlight %}
 
-Note that this filter rewrites the `group` key from each item. For instance, an item with such a group key:
+<p class="note  note--info"><strong>Note:</strong>  The `groupName` filter overrides the `group` key of each item.</p>
+
+Result of using the `group` key:
```
Comment (pascalduez) [inline extra-tools/index.md]: It's not the result of a filter, just the default data from SassDoc. 
This would need to be rewritten into something like "Dafault `group` key".

**Comment by pascalduez** [comment]
About "Result of using..." I'm fine with it, but we have to make sure to keep the distinction between the value before and after a filter is applied, which is sometimes not so clear in this PR.

**Comment by oxocode** [comment]
@pascalduez @valeriangalliat 

I spent some time and thought about this a little more. I've pushed an update that hopefully provides more clarity using the groupName filter through distinction of the before and after.

**Comment by valeriangalliat** [comment]
LGTM, what do you think @pascalduez?
Also we never had @HugoGiraudel's opinion here.

**Comment by pascalduez** [comment]
Wow, perfect. Thanks a lot @oxocode for taking the time to improve our docs, much appreciated!

So :+1:

**Comment by HugoGiraudel** [comment]
What is there to say, it’s great. Thank you so much. :)
